shared/js/hamburger.js

```diff
@@ -0,0 +1,32 @@
+$(function () {
```

Comment:
I've now moved this JS (and the really old JS that ajaxes the user menu into the header) over to the apps pipeline.